utd/parser.js

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env node
+
+'use strict';
+const Parser = require('../.lib/parser.js');
+
+/**
+ * Recognizes the accesses to the platform UpToDate
+ * @param  {Object} parsedUrl an object representing the URL to analyze
+ *                            main attributes: pathname, query, hostname
+ * @param  {Object} ec        an object representing the EC whose URL is being analyzed
+ * @return {Object} the result
+ */
+module.exports = new Parser(function analyseEC(parsedUrl, ec) {
+  let result = {};
+  let path   = parsedUrl.pathname;
+  let param  = parsedUrl.query || {};
+
+  let match;
+
+  if ((match = /^\/contents\/table-of-contents\/([a-z0-9-]+)\/?([a-z0-9-]+)?/i.exec(path)) !== null) {
```

Comment:
Something like `([a-z0-9-]+)(?:\/([a-z0-9-]+))?\/?` would be less likely to get side effects